addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -1040,6 +1041,12 @@ public void run() {
                                                     dateFormatter.format(currentTimeStamp),
                                                     dateFormatter.format(systemTimeStamp));
                                         }
+                                        if (systemTimeStamp - currentTimeStamp > EVENT_TIMESTAMP_MAX_DELTA) {
+                                            if (logger.isTraceEnabled()) {
```

Comment:
you only need this if there is logic involved in constructing the input for the logging statement, the actual trace call will do its own check whether the trace level is active and terminate asap if the level does not apply.